### ISSUE TITLE
Mac issue regarding the TP problem

### ISSUE THREAD

**Issue by EVBAS** [bluemagic123/tModLoader#4314]
### Version

Latest 1.4.4-stable

### OS

Mac

### Platform

Steam

### This bug affects...

Mod capability as a Modder

### Description

When the player,containing magic mirroror other transport items,is using any items,player will be tranported to the spwanpoint.The bug 
https://github.com/user-attachments/assets/31641e39-eb68-4b2b-a8cf-f883b9cabe07

**Comment by JavidPack** [comment]
Can you confirm this issue with 0 mods enabled? Please make sure that this issue happens with 0 mods enabled to confirm that it is a tModLoader bug and not an issue with a mod you have enabled.

If you can confirm that, post the logs.

**Comment by EVBAS** [comment]
yup,the error is stilled existed in environment with zero mod.(macOS)

**Comment by JavidPack** [comment]
Well, I can't replicate your issue myself, and you didn't post the logs so I can't take this any further. If you have more specific instructions on how to replicate I can try again.

**Comment by JavidPack** [comment]
This issue has either no recent activity or no reproducible steps, so it is being closed. We assume this issue has solved itself in some manner. Feel free to open a new issue if the issue persists and fill out the issue template completely.
